_commands/tools/pdftk.md

```diff
@@ -0,0 +1,45 @@
+---
+---
+
+pdftk
+---
+'pdftk' is a command that is used to manipulate Portable Document Format (PDF) files. There are many ways to manipulate PDFs through the command such as merging, splitting, encrypting and much more. 
+<!-- one line explanation would go here -->
+
+<!-- minimal example -->
+~~~bash
+$ pdftk in1.pdf in2.pdf cat output combinedIn.pdf
+~~~
+
+<!--more-->
+
+### Useful Options / Examples
+
+#### 'pdftk locked.pdf input_pw YouShallPass output unlocked.pdf'
```

Comment:
Don't just have the command as the title, have a short description of what it does, eg 'Decrypt PDF'